core/spec.md

```diff
@@ -3465,6 +3469,13 @@ the model update requests generate an error
 ([model_compliance_error](#model_compliance_error)) if existing entities are
 not compliant.
 
+For the purposes of validating that the existing entities in the Registry are
+compliant with the model, the mechanisms used to define the model (e.g.
+`$include` vs `ximportresources` vs defined locally) MUST NOT impact that
+analysis. In other words, model updates that have no semantic changes but
+rather switch between one of those 3 mechanisms would not invalidate any
```

Comment:
"must not" or "cannot" rather than "would not" perhaps? (It's not quite clear to me at the moment whether this is stating something that is inherently true, or specifying a rule that a server must follow.)

Reply:
went with MUST NOT.

It's making sure that a server doesn't look at these types of changes as "delete group X + add group X" and in the process ends up deleting all instances of that group. Treating an edit of an existing group should be obvious that it's not, but switching from "ximport" to "inlining" the definition might be viewed as something more drastic to some people, and we need to stop that thinking.